ISL.Providers.Storages.AzureBlobStorage/Services/Foundations/Files/StorageService.Validations.cs

```diff
@@ -49,12 +50,28 @@ private static void ValidateStorageArgumentsOnGetDownloadLink(
                 (Rule: IsInvalid(expiresOn), Parameter: "ExpiresOn"));
         }
 
+        private static void ValidateStorageArgumentsOnCreateAccessPolicy(
+            string container, List<string> policyNames)
+        {
+            Validate(
+                (Rule: IsInvalid(container), Parameter: "Container"),
+                (Rule: IsInvalidList(policyNames), Parameter: "PolicyNames"));
+
+            ValidatePolicyNames(policyNames);
+        }
+
         private static dynamic IsInvalid(string text) => new
         {
             Condition = String.IsNullOrWhiteSpace(text),
             Message = "Text is invalid"
         };
 
+        private static dynamic IsInvalidList(List<string> textList) => new
+        {
+            Condition = textList is null || textList.Contains("") || textList.Contains(" "),
```

Comment:
IsNullOrWhiteSpace have more than just "" or " ".  It would be better to use this so you get the same comparison as you would get if you use the method above this one.

```suggestion
            Condition = textList is null || textList.Any(string.IsNullOrWhiteSpace),
```

Reply:
This has been updated